### PR TITLE
added StretchableButton

### PR DESCRIPTION
**Pull request by Magduri**
video: 
https://user-images.githubusercontent.com/63363693/115647898-9aede780-a2f2-11eb-9df8-a29430a50be5.mp4

### CODE REVIEW

**Comment by Magduri** [comment]
mobile: 
screencapture-localhost-6006-2021-04-21-22_43_20

File: src/Containers/StretchableButton/StretchableButton.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+export const StretchableButton: React.FC<IStretchableButtonProps> = ({
+    buttonText,
+    minWidth,
+    isDraggable,
+    defaultHeight,
+    defaultPositionX,
+    defaultPositionY
+}) => {
+    return (
+            <Rnd disableDragging={!isDraggable} minWidth={minWidth} bounds="parent"
```
Comment (ralph-dev) [inline src/Containers/StretchableButton/StretchableButton.tsx:24]: Missing ...props

**Review by ralph-dev — APPROVED** [review]
Lgtm but you forgot the ...props now I cannot add margin to the component / anything else thus making it not extensible let's schedule a call once that is fixed

**Comment by Magduri** [comment]
https://user-images.githubusercontent.com/63363693/115962490-aa835100-a4e9-11eb-93f6-22b5aaeaa857.mp4

**Comment by ralph-dev** [comment]
looks good to me feel free to merge. / let's jump on a call

**Pull request merged**